docs/synapse/devguides/cell_permissions.rst

```diff
@@ -0,0 +1,331 @@
+*layer.lift*
+    Lift data from any layer.
+
+*layer.lift.<iden>*
+    Lift data from a specific layer.
+
+*layer.set.<name>*
+    Set the Layer definition for a Layer.
+
+*layer.write.<iden>*
+    Write to any layer.
+
+*layer.write.<iden>*
+    Write to a specific layer.
+
+*lib.telepath.open*
+    Open a Telepath Proxy to a URL.
+
+*lib.telepath.open.<scheme>*
+    Open a Telepath Proxy to a URL with a specific scheme.
+
+*model.prop.add.<form>*
+    Add an extended property to a form.
+
+*model.prop.del.<form>*
+    Remove an extended property from a form.
+
+*model.tagprop.add*
+    Add a tag property.
+
+*model.tagprop.del*
+    Remove a tag property.
+
+*model.univ.add*
+    Add an extended universal property.
+
+*model.univ.del*
+    Remove an extended universal property.
+
+*node.add*
+    Add any form of node.
+
+*node.add.<form>*
+    Add a specific form of node.  (ex. ``node.add.inet:ipv4``)
+
+*node.data.get*
+    Get the value of any node data property on a node.
+
+*node.data.get.<name>*
+    Get the value of a specific node data property on a node.
+
+*node.data.list*
+    List all of the node data properties on a node.
+
+*node.data.pop*
+    Remove and return the value of any node data property on a node.
+
+*node.data.pop.<name>*
+    Remove and return the value of a specific node data property on a node.
+
+*node.data.set*
+    Set any node data property on a node.
+
+*node.data.set.<name>*
+    Set a specific node data property on a node.
+
+*node.del*
+    Delete any form of node.
+
+*node.del.<form>*
+    Delete a <form> node. (ex. ``node.del.inet:ipv4``)
+
+*node.edge.add*
+    Add lightweight edges.
+
+*node.edge.add.<verb>*
+    Add lightweight edges with a specific verb.
+
+*node.edge.del*
+    Remove lightweight edges.
+
+*node.edge.del.<verb>*
+    Remove lightweight edges with a specific verb.
+
+*node.prop.del*
+    Delete any property.
+
+*node.prop.del.<prop>*
+    Delete a specific property.  (ex. ``node.prop.del.inet:ipv4:loc``)
+
+*node.prop.set*
+    Set any property.
+
+*node.prop.set.<prop>*
+    Set a specific property.  (ex. ``node.prop.set.inet:ipv4:loc``)
+
+*node.tag.add*
+    Add any tag to a node.
+
+*node.tag.add.<tag>*
+    Add a specific tag or subtag to a node. (ex. ``node.tag.add.foo.bar``)
+
+*node.tag.del*
+    Remove any tag from a node.
+
+*node.tag.del.<tag>*
+    Remove a specific tag or subtag to a node. (ex. ``node.tag.del.foo.bar``)
+
+*pkg.add*
+    Add a Storm package.
+
+*pkg.del*
+    Remove a Storm package.
+
+*queue.add*
+    Add a Queue.
+
+*queue.del*
+    Delete a Queue.
+
+*queue.get*
+    Get a Queue object.
+
+*queue.put*
+    Put an object in a Queue.
+
+*service.add*
+    Add a Storm Service.
+
+*service.del*
+    Remove a Storm Service.
+
+*service.get*
+    Get any Storm Service definition.
+
+*service.get.<name>*
+    Get a specific Storm Service definition.
+
+*service.list*
+    List the Storm Service definitions.
+
+*status*
+    Get status information for a Cortex.
```

Comment:
Note: This is currently a deprecated permission.

Reply:
note added